packages/angular/README.md

```diff
@@ -0,0 +1,118 @@
+<p align="center">
+  <img src="./src/assets/logo.png" width="150px" /> 
+</p>
+
+<h1 align="center">React dApp Starter Kit</h1>
+
+<p align="center">
+  This is a starter-kit for creating a <strong>dApp</strong> on the <strong>nOS</strong> platform
+</p>
+
+<p align="center">
+  <a href="https://github.com/nos/create-nos-dapp/releases">
+    <img src="https://img.shields.io/github/tag/nos/create-nos-dapp.svg?style=flat">
+  </a>
+  <a href='https://github.com/prettier/prettier'>
+    <img src='https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat'>
+  </a>
+</p>
+
+## Getting started
+To get started, you can check out the nOS [API documentation](https://github.com/nos/client/blob/develop/docs/api.md)
+
+## Purpose
+The goal of this project is to provide a quickstart for creating nOS dApps with Angular and various dependencies already installed and configured.
+
+In this repo you can find NEO specific tooling:
+* nOS functions exposed in a store
+* neon-js, providing extra NEO functionality
+
+Also general frameworks and tooling:
+* Angular 5
+* SCSS
+* Karma and Jasmine
+* Webpack
```

Comment:
Did you try out the angular parcel plugin? If it provides the exact functionality webpack does, it might be nice to use Parcel so we have a uniform toolchain

Reply:
I haven't. I will check it and if I can transfer all Webpack functionality to Parcel, I'll submit another commit. If there is something that is stopping me to transfer the things, I'll talk with you on Discord.